R/init-package.R

```diff
@@ -91,6 +91,9 @@
     }
   }
 
+  # Fix macOS SQLite database if needed (first load only)
+  .fixMacOSDatabaseIfNeeded(filePathFor("PKSimDB.sqlite"))
+
```

Comment:
I don't quite understand the workflow.
The conversion should take place only once after the package is installed.
But this will be called every time when the library is loaded in R session... or not?